queries.sql

```diff
@@ -24,3 +24,54 @@ WHERE name NOT IN ('Gabumon');
 -- Find all animals with a weight between 10.4kg and 17.3kg (inclusive)
 SELECT * FROM animals
 WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
+
+-- update species
+	BEGIN;
+	UPDATE animals
+	SET species = 'digimon'
+	WHERE species IS NULL AND name LIKE '%mon';
+	UPDATE animals
+	SET species = 'pokemon'
+	WHERE species IS NULL AND name NOT LIKE '%mon';
```

Comment:
- [ ] Please make sure to set the `species` column in the animals' table to `unspecified`, then rollback the transaction to ensure the changes were made and rollbacked. 🙏